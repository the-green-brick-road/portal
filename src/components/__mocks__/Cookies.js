/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Image component mock
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @24 may 2023
# Latest revision: 24 may 2023
# ---------------------------------------------------- */

/* Material UI includes */
import { Box, Typography }             from '@mui/material';

function Cookies(props) {

    /* --------- Gather inputs --------- */
    const { color } = props;

    /* -------- Defining theme --------- */
    return (
        <Box style={{ minHeight:'30px',zIndex:1000, position:'absolute', backgroundColor:color }} >
            <Typography> Cookies </Typography>
        </Box>
    );

}

export default Cookies;
