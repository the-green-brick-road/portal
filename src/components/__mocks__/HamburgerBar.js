/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# HamburgerBar component mock
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @24 may 2023
# Latest revision: 24 may 2023
# ---------------------------------------------------- */

/* Material UI includes */
import { Box, Typography }   from '@mui/material';

function HamburgerBar(props) {

    /* --------- Gather inputs --------- */
    const { height, isNegative, theme={} }  = props
    /* const componentName = 'HamburgerBar'; */

    /* -------- Defining theme --------- */
    let color = theme.palette.primary.main;
    if (isNegative) { color = theme.palette.common.white; }

    return (
        <Box style={{height:height, backgroundColor:color, width: '100%'}}>
            <Typography> HamburgerBar </Typography>
        </Box>
    );

}

export default HamburgerBar;
