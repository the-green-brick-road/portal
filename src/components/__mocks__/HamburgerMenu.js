/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# HamburgerMenu component mock
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @24 may 2023
# Latest revision: 24 may 2023
# ---------------------------------------------------- */

/* Material UI includes */
import { Box, Typography }   from '@mui/material';

function HamburgerMenu(props) {

    /* --------- Gather inputs --------- */
    const { top , margin, isNegative, theme={} } = props;
    /* const componentName = 'HamburgerBar'; */

    /* -------- Defining theme --------- */
    let color = theme.palette.primary.main;
    if (isNegative) { color = theme.palette.common.white; }
    const menuWidthString = `calc( 100vw - 2 * ${margin})`;

    return (
        <Box style={{ minHeight:'30px', width:menuWidthString, top:top, zIndex:1000, position:'absolute', backgroundColor:color }} >
            <Typography> HamburgerMenu </Typography>
        </Box>
    );

}

export default HamburgerMenu;