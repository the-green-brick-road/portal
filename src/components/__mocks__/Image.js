/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Image component mock
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @24 may 2023
# Latest revision: 24 may 2023
# ---------------------------------------------------- */

/* Material UI includes */
import { Box }             from '@mui/material';

function Image(props) {

    /* --------- Gather inputs --------- */
    const { style={} }    = props;

    const local_style = JSON.parse(JSON.stringify(style))

    if (!('backgroundColor' in local_style)) { local_style['backgroundColor'] = 'blue' }

    /* ----------- Define HTML --------- */
    /* eslint-disable padded-blocks */
    return (
        <Box style={local_style} />
    );
    /* eslint-enable padded-blocks */

}

export default Image;
