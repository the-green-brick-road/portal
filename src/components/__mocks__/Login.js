/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Login component mock
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @29 may 2023
# Latest revision: 29 may 2023
# ---------------------------------------------------- */

/* Material UI includes */
import { Box }             from '@mui/material';

function Login(props) {

    /* --------- Gather inputs --------- */
    const { style={} }    = props;

    const local_style = JSON.parse(JSON.stringify(style))

    if (!('backgroundColor' in local_style)) { local_style['backgroundColor'] = 'green' }

    /* ----------- Define HTML --------- */
    /* eslint-disable padded-blocks */
    return (
        <Box style={local_style} />
    );
    /* eslint-enable padded-blocks */

}

export default Login;
