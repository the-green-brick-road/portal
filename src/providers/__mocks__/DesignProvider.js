/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Design provider mock
# -------------------------------------------------------
# NadÃ¨ge LEMPERIERE, @09 may 2023
# Latest revision: 09 may 2023
# ---------------------------------------------------- */

/* React includes */
import { createContext }       from 'react';

/* Material UI includes */
import { CssBaseline }                                 from '@mui/material';
import { ThemeProvider, createTheme }                  from '@mui/material/styles';

const context = createContext(null);

/* ----- Define provider values ---- */
var state = {
    isWebpSupported: true,
    isSliding:       false,
    isDarkMode:      false,
    images:          {},
    screen:          '',
    sizes:           {},
    setIsSliding:    jest.fn((value) => { state.isSliding = value }),
};

function DesignProvider(props) {

    const { children, isWebpSupported, screen, isDarkMode = false, isSliding = false, sizes, images = [], theme = {} } = props;

    /* ------ Defining behaviour ------- */
    var local_theme = {}
    if (Object.keys(theme).length !== 0) { local_theme = createTheme(theme); }

    state.isWebpSupported = isWebpSupported;
    state.screen          = screen;
    state.images          = {}
    state.sizes           = sizes;
    state.isSliding       = isSliding;
    state.isDarkMode      = isDarkMode;
    for (const image of images)
    {

        const rawext = image["raw"].split('.').pop();
        const webext = image["web"].split('.').pop();

        state.images[image.name] = {}
        state.images[image.name][rawext] = {}
        state.images[image.name][webext] = {}

        state.images[image.name][rawext]['small']           = {}
        state.images[image.name][rawext]['small']['img']    = require(`../../assets/small/${image.raw}`)
        state.images[image.name][rawext]['small']['width']  = `${sizes['small-width']}w`
        state.images[image.name][rawext]['medium']          = {}
        state.images[image.name][rawext]['medium']['img']   = require(`../../assets/medium/${image.raw}`)
        state.images[image.name][rawext]['medium']['width'] = `${sizes['medium-width']}w`
        state.images[image.name][rawext]['large']           = {}
        state.images[image.name][rawext]['large']['img']    = require(`../../assets/${image.raw}`)
        state.images[image.name][rawext]['large']['width']  = `${sizes['large-width']}w`

        state.images[image.name][webext]['small']           = {}
        state.images[image.name][webext]['small']['img']    = require(`../../assets/small/${image.web}`)
        state.images[image.name][webext]['small']['width']  = `${sizes['small-width']}w`
        state.images[image.name][webext]['medium']          = {}
        state.images[image.name][webext]['medium']['img']   = require(`../../assets/medium/${image.web}`)
        state.images[image.name][webext]['medium']['width'] = `${sizes['medium-width']}w`
        state.images[image.name][webext]['large']           = {}
        state.images[image.name][webext]['large']['img']    = require(`../../assets/${image.web}`)
        state.images[image.name][webext]['large']['width']  = `${sizes['large-width']}w`

        state.images[image.name]['default']= require(`../../assets/${image.raw}`)

    }

    /* ----------- Define HTML --------- */
    return (
        <ThemeProvider theme={local_theme}>
            <CssBaseline />
            <context.Provider value={state}>
                {children}
            </context.Provider>
        </ThemeProvider>
    );

}

function useDesign() { return state; };

export { useDesign, DesignProvider };