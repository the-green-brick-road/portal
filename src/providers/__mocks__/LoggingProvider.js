/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Logging provider mock
# -------------------------------------------------------
# NadÃ¨ge LEMPERIERE, @09 may 2023
# Latest revision: 09 may 2023
# ---------------------------------------------------- */

/* React includes */
import { createContext }       from 'react';

const context = createContext(null);

/* ----- Define provider values ---- */
var state = {
    isLoggingActivated : true,
    logText: jest.fn((component, level, topic, message) => {  console.log(`${component} - ${level} - ${topic} -${message}`) }),
};

function LoggingProvider(props) {

    const { children } = props;

    /* ----------- Define HTML --------- */
    return (
        <context.Provider value={state}>
            {children}
        </context.Provider>
    );

}

function useLogging() { return state; };

export { useLogging, LoggingProvider };