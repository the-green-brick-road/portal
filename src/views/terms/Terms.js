/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Terms and Conditions view component
# -------------------------------------------------------
# Nad√®ge LEMPERIERE, @20 june 2023
# Latest revision: 20 june 2023
# ---------------------------------------------------- */

/* React includes */
import { Profiler }                                                                from 'react';

/* Material UI includes */
import { Typography, Container, List, ListItemText, ListItemIcon, ListItem } from '@mui/material';
import { default as Circle }                                                 from '@mui/icons-material/Circle';

/* Portal includes */
import { useDesign, useLogging }                                             from '../../providers';

function Terms() {

    /* --------- Gather inputs --------- */
    const { sizes }     = useDesign();
    const { onRender }  = useLogging();
    const componentName = 'Terms'

    /* ----------- Define HTML --------- */
    return (
        <Profiler id={componentName} onRender={onRender}>
            <Container style={{ backgroundColor:'rgba(255,255,255,0)', width:'100%', height:sizes['menu-height'], padding:0, position:'relative' }}/>
            <Container style={{ marginBottom:10 }}>
                <Typography variant="h1" style={{marginBottom:'30px'}}> Terms & Conditions </Typography>
                <Typography style={{fontSize:'12px'}}>These Terms and Conditions (the "Terms and Conditions") govern the use of the-green-brick-road.org. This Site is owned and operated by the-green-brick-road. This site is an educational website. </Typography>
                <Typography style={{fontSize:'12px'}}>By using this Site, you indicate that you have read and understood these Terms and Conditions and agree to abide by them at all times </Typography>
                <Typography variant="h2"> Intellectual property </Typography>
                <Typography style={{fontSize:'12px'}}>All content published and made available on our site is the property of the-green-brick-road. This includes, but is not limited to images, text, logos, documents, downloadable files and anything that contributes to the composition of our Site.</Typography>
                <Typography variant="h2"> Acceptable Use </Typography>
                <Typography style={{fontSize:'12px'}}>As a user of our Site, you agree to use our Site legally, not to use our Site for illegal purposes, and not to:</Typography>
                <List>
                    <ListItem style={{padding:0}}><ListItemIcon style={{minWidth:'8px'}}><Circle color="primary" style={{height:'8px',minWidth:'8px'}}/></ListItemIcon><ListItemText primaryTypographyProps={{fontSize:'12px'}}>Violate the rights of other users of our Site;</ListItemText></ListItem>
                    <ListItem style={{padding:0}}><ListItemIcon style={{minWidth:'8px'}}><Circle color="primary" style={{height:'8px',minWidth:'8px'}}/></ListItemIcon><ListItemText primaryTypographyProps={{fontSize:'12px'}}>Violate the intellectual property rights of the Site owners or any third party to the Site;</ListItemText></ListItem>
                    <ListItem style={{padding:0}}><ListItemIcon style={{minWidth:'8px'}}><Circle color="primary" style={{height:'8px',minWidth:'8px'}}/></ListItemIcon><ListItemText primaryTypographyProps={{fontSize:'12px'}}>Hack into the account of another user of the Site; or</ListItemText></ListItem>
                    <ListItem style={{padding:0}}><ListItemIcon style={{minWidth:'8px'}}><Circle color="primary" style={{height:'8px',minWidth:'8px'}}/></ListItemIcon><ListItemText primaryTypographyProps={{fontSize:'12px'}}>Act in any way that could be considered fraudulent.</ListItemText></ListItem>
                </List>
                <Typography style={{fontSize:'12px'}}>If we believe you are using our Site illegally or in a manner that violates these Terms and Conditions, we reserve the right to limit, suspend or terminate your access to our Site. We also reserve the right to take any legal steps necessary to prevent you from accessing our Site.</Typography>
                <Typography variant="h2"> Accounts </Typography>
                <Typography style={{fontSize:'12px'}}>When you create an account on our Site, you agree to the following:</Typography>
                <List>
                    <ListItem style={{padding:0}}><ListItemIcon style={{minWidth:'8px'}}><Circle color="primary" style={{height:'8px',minWidth:'8px'}}/></ListItemIcon><ListItemText primaryTypographyProps={{fontSize:'12px'}}>You are solely responsible for your account and the security and privacy of your account, including passwords or sensitive information attached to that account; and</ListItemText></ListItem>
                    <ListItem style={{padding:0}}><ListItemIcon style={{minWidth:'8px'}}><Circle color="primary" style={{height:'8px',minWidth:'8px'}}/></ListItemIcon><ListItemText primaryTypographyProps={{fontSize:'12px'}}>All personal information you provide to us through your account is up to date, accurate, and truthful and that you will update your personal information if it changes.</ListItemText></ListItem>
                </List>
                <Typography style={{fontSize:'12px'}}>We reserve the right to suspend or terminate your account if you are using our Site illegally or if you violate these Terms and Conditions.</Typography>
                <Typography variant="h2"> Links to Other Websites </Typography>
                <Typography style={{fontSize:'12px'}}>Our Site contains links to third party websites or services that we do not own or control. We are not responsible for the content, policies, or practices of any third party website or service linked to on our Site. It is your responsibility to read the terms and conditions and privacy policies of these third party websites before using these sites.</Typography>
                <Typography variant="h2"> Limitation of Liability </Typography>
                <Typography style={{fontSize:'12px'}}>our administrators will not be liable for any actions, claims, losses, damages, liabilities and expenses including legal fees from your use of the Site.</Typography>
                <Typography variant="h2"> Indemnity </Typography>
                <Typography style={{fontSize:'12px'}}>Except where prohibited by law, by using this Site you indemnify and hold harmless our directors, officers, agents, employees, subsidiaries, and affiliates from any actions, claims, losses, damages, liabilities and expenses including legal fees arising out of your use of our Site or your violation of these Terms and Conditions.</Typography>
                <Typography variant="h2"> Applicable Law </Typography>
                <Typography style={{fontSize:'12px'}}>These Terms and Conditions are governed by the laws of the State of New York.</Typography>
                <Typography variant="h2"> Severability </Typography>
                <Typography style={{fontSize:'12px'}}>If at any time any of the provisions set forth in these Terms and Conditions are found to be inconsistent or invalid under applicable laws, those provisions will be deemed void and will be removed from these Terms and Conditions. All other provisions will not be affected by the removal and the rest of these Terms and Conditions will still be considered valid.</Typography>
                <Typography variant="h2"> Changes </Typography>
                <Typography style={{fontSize:'12px'}}>These Terms and Conditions may be amended from time to time in order to maintain compliance with the law and to reflect any changes to the way we operate our Site and the way we expect users to behave on our Site. We will notify users by email of changes to these Terms and Conditions or post a notice on our Site.</Typography>
                <Typography variant="h2"> Contact Details </Typography>
                <Typography style={{fontSize:'12px'}}>Please contact us if you have any questions or concerns.</Typography>

            </Container>
        </Profiler>
    );

}


export default Terms;