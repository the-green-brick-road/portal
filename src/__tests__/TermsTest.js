/* ------------------------------------------------------
# THE GREEN BRICK ROAD
# -------------------------------------------------------
# Copyright (c) [2023] The Green Brick Road
# All rights reserved
# -------------------------------------------------------
# Terms & Conditions view component test suite
# -------------------------------------------------------
# NadÃ¨ge LEMPERIERE, @30 may 2023
# Latest revision: 30 may 2023
# ---------------------------------------------------- */

/* Test includes */
import { render, prettyDOM, act }  from '@testing-library/react'
import { expect, test}             from '@jest/globals';

/* Component under test */
import { default as Terms }        from '../views/terms/Terms';

/* Mocks includes */
/* eslint-disable jest/no-mocks-import */
import { useDesign as mockUseDesign, DesignProvider as MockDesignProvider } from '../providers/__mocks__/DesignProvider';
import { useLogging as mockUseLogging, LoggingProvider as MockLoggingProvider } from '../providers/__mocks__/LoggingProvider';
jest.mock('../providers', () => ({
    useDesign: (() => { return mockUseDesign(); }),
    useLogging: (() => { return mockUseLogging() }),
}));
/* eslint-enable jest/no-mocks-import */

describe("Blog view" ,() => {

    const sizes = {
        "small-width"          : 200,
        "medium-width"         : 400,
        "large-width"          : 800,
        "menu-height"          : 80,
        "margin"               : 10,
        "hamburger-height"     : 40,
    }

    const theme = {
        "palette": {
            "common": {
                "black": "#444444",
                "white": "#555555",
            },
            "primary": {
                "main": "#666666",
                "light": "#666667",
                "dark": "#666668",
            },
            "secondary" : {
                "main": "#777777",
                "light": "#777778",
                "dark": "#777779",
            },
            "disabled" : {
                "main": "#888888",
                "light": "#888889",
                "dark": "#888880",
            },
            "text" : {
                "primary": "#111111",
                "secondary": "#222222",
                "disabled": "#333333",
            },
        },
        "direction": "ltr",
        "typography": {
            "h1": {
                "fontSize": "2.5rem",
                "color": "#111111",
                "textTransform": "uppercase",
                "fontFamily": "Roboto Condensed",
            },
            "h2": {
                "fontSize": "2rem",
                "color": "#222222",
                "textTransform": "lowercase",
                "fontFamily": "Roboto Condensed",
            },
            "body1": {
                "fontSize": "14px",
                "color": "black",
                "fontFamily": "Roboto",
                "position": "relative",
                "textAlign": "justify",
            },
        },
    }

    test('Should display terms and conditions page', async () => {

        const view = render(
            <div>
                <MockLoggingProvider>
                    <MockDesignProvider screen='large' theme={theme} sizes={sizes}>
                        <Terms/>
                    </MockDesignProvider>
                </MockLoggingProvider>
            </div>
        );
        const tree = prettyDOM(view.baseElement, Number.POSITIVE_INFINITY, {filterNode: () => true, escapeString: false, highlight: false});
        expect(tree).toMatchSnapshot();

    })

})
