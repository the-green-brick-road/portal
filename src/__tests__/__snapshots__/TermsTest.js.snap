// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog view Should display terms and conditions page 1`] = `
"<body>
  <div
    id=\\"root\\"
  />
  <div>
    <div>
      <div
        class=\\"MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root\\"
        style=\\"background-color: rgba(255, 255, 255, 0); width: 100%; height: 80px; padding: 0px; position: relative;\\"
      />
      <div
        class=\\"MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root\\"
        style=\\"margin-bottom: 10px;\\"
      >
        <h1
          class=\\"MuiTypography-root MuiTypography-h1 css-wfxni2-MuiTypography-root\\"
          style=\\"margin-bottom: 30px;\\"
        >
           Terms & Conditions 
        </h1>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          These Terms and Conditions (the \\"Terms and Conditions\\") govern the use of the-green-brick-road.org. This Site is owned and operated by the-green-brick-road. This site is an educational website. 
        </p>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          By using this Site, you indicate that you have read and understood these Terms and Conditions and agree to abide by them at all times 
        </p>
        <h2
          class=\\"MuiTypography-root MuiTypography-h2 css-11eu52g-MuiTypography-root\\"
        >
           Intellectual property 
        </h2>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          All content published and made available on our site is the property of the-green-brick-road. This includes, but is not limited to images, text, logos, documents, downloadable files and anything that contributes to the composition of our Site.
        </p>
        <h2
          class=\\"MuiTypography-root MuiTypography-h2 css-11eu52g-MuiTypography-root\\"
        >
           Acceptable Use 
        </h2>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          As a user of our Site, you agree to use our Site legally, not to use our Site for illegal purposes, and not to:
        </p>
        <ul
          class=\\"MuiList-root MuiList-padding css-h4y409-MuiList-root\\"
        >
          <li
            class=\\"MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1os9ulk-MuiListItem-root\\"
            style=\\"padding: 0px;\\"
          >
            <div
              class=\\"MuiListItemIcon-root css-cveggr-MuiListItemIcon-root\\"
              style=\\"min-width: 8px;\\"
            >
              <svg
                aria-hidden=\\"true\\"
                class=\\"MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-trob2m-MuiSvgIcon-root\\"
                data-testid=\\"CircleIcon\\"
                focusable=\\"false\\"
                style=\\"height: 8px; min-width: 8px;\\"
                viewBox=\\"0 0 24 24\\"
              >
                <path
                  d=\\"M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2\\"
                />
              </svg>
            </div>
            <div
              class=\\"MuiListItemText-root css-tlelie-MuiListItemText-root\\"
            >
              <span
                class=\\"MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-1ml28qw-MuiTypography-root\\"
              >
                Violate the rights of other users of our Site;
              </span>
            </div>
          </li>
          <li
            class=\\"MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1os9ulk-MuiListItem-root\\"
            style=\\"padding: 0px;\\"
          >
            <div
              class=\\"MuiListItemIcon-root css-cveggr-MuiListItemIcon-root\\"
              style=\\"min-width: 8px;\\"
            >
              <svg
                aria-hidden=\\"true\\"
                class=\\"MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-trob2m-MuiSvgIcon-root\\"
                data-testid=\\"CircleIcon\\"
                focusable=\\"false\\"
                style=\\"height: 8px; min-width: 8px;\\"
                viewBox=\\"0 0 24 24\\"
              >
                <path
                  d=\\"M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2\\"
                />
              </svg>
            </div>
            <div
              class=\\"MuiListItemText-root css-tlelie-MuiListItemText-root\\"
            >
              <span
                class=\\"MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-1ml28qw-MuiTypography-root\\"
              >
                Violate the intellectual property rights of the Site owners or any third party to the Site;
              </span>
            </div>
          </li>
          <li
            class=\\"MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1os9ulk-MuiListItem-root\\"
            style=\\"padding: 0px;\\"
          >
            <div
              class=\\"MuiListItemIcon-root css-cveggr-MuiListItemIcon-root\\"
              style=\\"min-width: 8px;\\"
            >
              <svg
                aria-hidden=\\"true\\"
                class=\\"MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-trob2m-MuiSvgIcon-root\\"
                data-testid=\\"CircleIcon\\"
                focusable=\\"false\\"
                style=\\"height: 8px; min-width: 8px;\\"
                viewBox=\\"0 0 24 24\\"
              >
                <path
                  d=\\"M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2\\"
                />
              </svg>
            </div>
            <div
              class=\\"MuiListItemText-root css-tlelie-MuiListItemText-root\\"
            >
              <span
                class=\\"MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-1ml28qw-MuiTypography-root\\"
              >
                Hack into the account of another user of the Site; or
              </span>
            </div>
          </li>
          <li
            class=\\"MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1os9ulk-MuiListItem-root\\"
            style=\\"padding: 0px;\\"
          >
            <div
              class=\\"MuiListItemIcon-root css-cveggr-MuiListItemIcon-root\\"
              style=\\"min-width: 8px;\\"
            >
              <svg
                aria-hidden=\\"true\\"
                class=\\"MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-trob2m-MuiSvgIcon-root\\"
                data-testid=\\"CircleIcon\\"
                focusable=\\"false\\"
                style=\\"height: 8px; min-width: 8px;\\"
                viewBox=\\"0 0 24 24\\"
              >
                <path
                  d=\\"M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2\\"
                />
              </svg>
            </div>
            <div
              class=\\"MuiListItemText-root css-tlelie-MuiListItemText-root\\"
            >
              <span
                class=\\"MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-1ml28qw-MuiTypography-root\\"
              >
                Act in any way that could be considered fraudulent.
              </span>
            </div>
          </li>
        </ul>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          If we believe you are using our Site illegally or in a manner that violates these Terms and Conditions, we reserve the right to limit, suspend or terminate your access to our Site. We also reserve the right to take any legal steps necessary to prevent you from accessing our Site.
        </p>
        <h2
          class=\\"MuiTypography-root MuiTypography-h2 css-11eu52g-MuiTypography-root\\"
        >
           Accounts 
        </h2>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          When you create an account on our Site, you agree to the following:
        </p>
        <ul
          class=\\"MuiList-root MuiList-padding css-h4y409-MuiList-root\\"
        >
          <li
            class=\\"MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1os9ulk-MuiListItem-root\\"
            style=\\"padding: 0px;\\"
          >
            <div
              class=\\"MuiListItemIcon-root css-cveggr-MuiListItemIcon-root\\"
              style=\\"min-width: 8px;\\"
            >
              <svg
                aria-hidden=\\"true\\"
                class=\\"MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-trob2m-MuiSvgIcon-root\\"
                data-testid=\\"CircleIcon\\"
                focusable=\\"false\\"
                style=\\"height: 8px; min-width: 8px;\\"
                viewBox=\\"0 0 24 24\\"
              >
                <path
                  d=\\"M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2\\"
                />
              </svg>
            </div>
            <div
              class=\\"MuiListItemText-root css-tlelie-MuiListItemText-root\\"
            >
              <span
                class=\\"MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-1ml28qw-MuiTypography-root\\"
              >
                You are solely responsible for your account and the security and privacy of your account, including passwords or sensitive information attached to that account; and
              </span>
            </div>
          </li>
          <li
            class=\\"MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1os9ulk-MuiListItem-root\\"
            style=\\"padding: 0px;\\"
          >
            <div
              class=\\"MuiListItemIcon-root css-cveggr-MuiListItemIcon-root\\"
              style=\\"min-width: 8px;\\"
            >
              <svg
                aria-hidden=\\"true\\"
                class=\\"MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-trob2m-MuiSvgIcon-root\\"
                data-testid=\\"CircleIcon\\"
                focusable=\\"false\\"
                style=\\"height: 8px; min-width: 8px;\\"
                viewBox=\\"0 0 24 24\\"
              >
                <path
                  d=\\"M12 2C6.47 2 2 6.47 2 12s4.47 10 10 10 10-4.47 10-10S17.53 2 12 2\\"
                />
              </svg>
            </div>
            <div
              class=\\"MuiListItemText-root css-tlelie-MuiListItemText-root\\"
            >
              <span
                class=\\"MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-1ml28qw-MuiTypography-root\\"
              >
                All personal information you provide to us through your account is up to date, accurate, and truthful and that you will update your personal information if it changes.
              </span>
            </div>
          </li>
        </ul>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          We reserve the right to suspend or terminate your account if you are using our Site illegally or if you violate these Terms and Conditions.
        </p>
        <h2
          class=\\"MuiTypography-root MuiTypography-h2 css-11eu52g-MuiTypography-root\\"
        >
           Links to Other Websites 
        </h2>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          Our Site contains links to third party websites or services that we do not own or control. We are not responsible for the content, policies, or practices of any third party website or service linked to on our Site. It is your responsibility to read the terms and conditions and privacy policies of these third party websites before using these sites.
        </p>
        <h2
          class=\\"MuiTypography-root MuiTypography-h2 css-11eu52g-MuiTypography-root\\"
        >
           Limitation of Liability 
        </h2>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          our administrators will not be liable for any actions, claims, losses, damages, liabilities and expenses including legal fees from your use of the Site.
        </p>
        <h2
          class=\\"MuiTypography-root MuiTypography-h2 css-11eu52g-MuiTypography-root\\"
        >
           Indemnity 
        </h2>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          Except where prohibited by law, by using this Site you indemnify and hold harmless our directors, officers, agents, employees, subsidiaries, and affiliates from any actions, claims, losses, damages, liabilities and expenses including legal fees arising out of your use of our Site or your violation of these Terms and Conditions.
        </p>
        <h2
          class=\\"MuiTypography-root MuiTypography-h2 css-11eu52g-MuiTypography-root\\"
        >
           Applicable Law 
        </h2>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          These Terms and Conditions are governed by the laws of the State of New York.
        </p>
        <h2
          class=\\"MuiTypography-root MuiTypography-h2 css-11eu52g-MuiTypography-root\\"
        >
           Severability 
        </h2>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          If at any time any of the provisions set forth in these Terms and Conditions are found to be inconsistent or invalid under applicable laws, those provisions will be deemed void and will be removed from these Terms and Conditions. All other provisions will not be affected by the removal and the rest of these Terms and Conditions will still be considered valid.
        </p>
        <h2
          class=\\"MuiTypography-root MuiTypography-h2 css-11eu52g-MuiTypography-root\\"
        >
           Changes 
        </h2>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          These Terms and Conditions may be amended from time to time in order to maintain compliance with the law and to reflect any changes to the way we operate our Site and the way we expect users to behave on our Site. We will notify users by email of changes to these Terms and Conditions or post a notice on our Site.
        </p>
        <h2
          class=\\"MuiTypography-root MuiTypography-h2 css-11eu52g-MuiTypography-root\\"
        >
           Contact Details 
        </h2>
        <p
          class=\\"MuiTypography-root MuiTypography-body1 css-15v4sip-MuiTypography-root\\"
          style=\\"font-size: 12px;\\"
        >
          Please contact us if you have any questions or concerns.
        </p>
      </div>
    </div>
  </div>
</body>"
`;
